src/model/include/iDynTree/Model/Model.h

```diff
@@ -295,18 +295,24 @@ namespace iDynTree
                                    const std::string & newLinkName, Link & newLink);
 
         /**
-         * Insert a joint to the model, and add also a link. The other link in the existing joint
-         * that is not the unmovalbe link is the movable link.
-         * The new link replace the movable link as an attached link to the existing joint.
-         * Then a new joint is added between the new link and the movable link.
+         * \brief Displace a link by inserting a new link and a new joint.
+         * Displace a link by replacing it with a new link in the existing joint and insert new joint between the new link and the displaced link.
+         * Inputs:
+         * existingJoint is where the new link will be inserted.
```

Comment:
The syntrax for Doxygen parameters is `@param[in]` for all the input parameters (in this case it would be:
~~~
* @param[in] existingJoint is where the new link will be inserted. 
* @param[in] nmovableLink is the link in the existingJoint that will remain the same. 
~~~
and so on and so forth, see https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam .

Reply:
Thanks Ill do the change